tests/SignedClosureTest.php

```diff
@@ -119,11 +115,10 @@ public function testInvalidSecuredClosureWithoutSecuriyProvider()
         unserialize($value);
     }
 
-    /**
-     * @expectedException \Opis\Closure\SecurityException
-     */
     public function testInvalidJsonSecuredClosureWithoutSecuriyProvider()
     {
+        $this->expectException(SecurityException::class);
```

Comment:
You'll need to do method exists checks so you can support the old versions of PHPUnit.